tests/Microsoft.Bot.Builder.LUIS.Tests/LuisRecognizerTests.cs

```diff
@@ -17,17 +19,29 @@ namespace Microsoft.Bot.Builder.LUIS.Tests
      */
     public class LuisRecognizerTests
     {
+
+        private readonly string _luisAppId = TestUtilities.GetKey("LUISAPPID");
+        private readonly string _subscriptionKey = TestUtilities.GetKey("LUISAPPKEY");
+        private readonly string _luisUriBase = TestUtilities.GetKey("LUISURIBASE");
+
+
         [TestMethod]
         public async Task SingleIntent_SimplyEntity()
         {
+            if (_luisAppId == null || _subscriptionKey == null || _luisUriBase == null)
```

Comment:
This if statment shows up in several places. Consider making a method.

Reply:
Done